### ISSUE TITLE
mdtest versioning scheme should be updated

### ISSUE THREAD

**Issue by glennklockwood** [hpc/ior#128]
It looks like mdtest still maintains its own versioning scheme despite being integrated into the IOR source tree, e.g.,

```
$ mpirun -np 1 /home/glock/ior/test/_inst/bin/mdtest -n 1 -d .
-- started at 01/18/2019 15:36:58 --

mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)
```

Should we change the mdtest version to match the IOR version going forward since they are released together?  Otherwise, changes to the backend code that prompt an IOR version uptick could affect mdtest unintentionally.

**Comment by johnbent** [comment]
Is this related to #132 ?  I think so.  Once we fix #132 then I think this one might go away.

**Comment by JulianKunkel** [comment]
The issue has been fixed a while ago by using the same version from META in both IOR and MDTest
